gems/sorbet/test/snapshot/partial/db_schema/src/db/schema.rb

```diff
@@ -0,0 +1,3 @@
+
+# This should be skipped by `srb init`
+raise "Shouldn't be required"
```

Comment:
I think we're going to have to try harder to observe a side effect here. I don't think this test fails before your fix is included, because we swallow load errors in an attempt to recover and plow forward.

I think what I would try to do is make db/schema.rb delete a file in `src` that would otherwise show up in the expected folder. Like create `src/sorbet/foo.txt`, have `db/schema.rb` delete this file, and assert that it `expected/sorbet/foo.txt` shows up after running the test.

Reply:
OK, I've changed it to remove `sorbet/foo.txt` when `db/schema.rb` is required, and validated that commenting out the new line in `require_everything.rb` causes a test failure.